jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/PluginConfigurationProcessor.java

```diff
@@ -38,8 +37,7 @@
 /** Configures and provides builders for the image building goals. */
 class PluginConfigurationProcessor {
 
-  /** {@code User-Agent} header suffix to send to the registry. */
-  private static final String USER_AGENT_SUFFIX = "jib-maven-plugin";
+  private static final String TOOL_NAME = "jib-maven-plugin";
```

Comment:
Hmm, actually it seems a bit out-of-place to be in the `PluginConfigurationProcessor` - it's actually already defined in `MavenProjectProperties`.

Reply:
In gradle, the project properties plugin name is just `jib`. Should we change it to `jib-gradle-plugin`?

Reply:
Oh right, so that was named `jib` because the configuration is applied to `jib` (the extension). Perhaps we should have a separate constant for the tool name then.